packages/terra-i18n-plugin/tests/i18n-plugin-spec.test.js

```diff
@@ -103,6 +107,45 @@ describe('i18n-aggregator-plugin', () => {
     });
   });
 
+  describe('when searching directories for translation messages with options inputFileSystem given', () => {
+    const compiler = createCompiler();
+    const inputFileSystem = createInputFileSystem();
+    const inputReadSourceDirectories = [];
+    const inputReadFileSyncPaths = [];
+    const inputReadFileEncondings = [];
+    let SpyOnInputReadDirSync;
+    let SpyOnInputReadFileSync;
+    beforeAll(() => {
+      SpyOnInputReadDirSync = spyOn(inputFileSystem, 'readdirSync').and.callFake((dirName) => {
+        inputReadSourceDirectories.push(dirName);
+      });
+      SpyOnInputReadFileSync = spyOn(inputFileSystem, 'readFileSync').and.callFake((filePath, encoding) => {
+        inputReadFileSyncPaths.push(filePath);
+        inputReadFileEncondings.push(encoding);
+      });
+
+      new I18nAggregatorPlugin({
+        baseDirectory,
+        supportedLocales,
+        inputFileSystem,
+      }).apply(compiler);
+      compiler._plugins['after-environment'][0]();
```

Comment:
Is this line a test specific line or does this require additional configuration to the I18nAggregatorPlugin?

Reply:
This is needed to have the test hit the 'after-environment' hook in the code.  Else, it would skip over the following line of code.  Is how I solved it the when I updated this to be able to provide your own outputFileSystem by calling the plugin directly.
"compiler.plugin('after-environment', () => {"

Reply:
Okay. I assumed that was the case but wanted to know for sure!